ftw/upgrade/tests/test_upgrade_step_intids.py

```diff
@@ -0,0 +1,7 @@
+from ftw.upgrade.testing import INTID_UPGRADE_FUNCTIONAL_TESTING
+from ftw.upgrade.tests.test_upgrade_step import TestUpgradeStep
+
+
+class TestUpgradeStepIntids(TestUpgradeStep):
```

Comment:
yes, in two ways:
- that code has been used in production to fix an "failed" migration, the fix script can be found in [opengever.maintenance/scripts/fix_mail_class_intids.py](https://github.com/4teamwork/opengever.maintenance/blob/master/opengever/maintenance/scripts/fix_mail_class_intids.py#L36-L45)
- i've written a test-migration in `opengever.core` to confirm that an `intids.getObject(..)` lookup (directly after starting the zope process) returns a instance of the old class without the fix, and that the class is correct when the fix is enabled

so i am quite certain that the fix should work, or at the very least doesn't break stuff :sweat:.